tests/system/plugin-reachable-vulns.test.ts

```diff
@@ -68,6 +79,9 @@ test('inspect on test-project pom with reachable vulns', async (t) => {
     javaCallGraphBuilderStub.calledWith(testProjectPath),
     'call graph builder was called with the correct path',
   );
+  t.ok(callGraphMetrics.calledOnce, 'callgraph metrics were fetched');
+  t.equals((result.plugin.meta as any).callGraphMetrics, metrics);
```

Comment:
Wonder if you can use `?.` here instead of as any

Reply:
the `?.` in only for union types with `undefined`. The proper thing would be to add the `Metrics` type from the `callgraph-builder` to the meta type.